src/Hl7.Fhir.ElementModel.Tests/ElementNodeTests.cs

```diff
@@ -285,13 +285,34 @@ public void CannotUseAbstractType()
             try
             {
                 var typedEntry =
-                    entry.ToTypedElement(provider, "Element");
+                    entry.ToTypedElement(_provider, "Element");
                 Microsoft.VisualStudio.TestTools.UnitTesting.Assert.Fail("Should have thrown on invalid Div format");
             }
             catch (ArgumentException)
             {
             }
         }
 
+        [TestMethod]
+        public void CreateFromPrimitive()
```

Comment:
Should this test not be in the fhir-net-common project?

Reply:
Ik denk het van wel.